bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Display.java

```diff
@@ -5152,16 +5152,90 @@ String wrapText (String text, long handle, int width) {
 }
 
 static String withCrLf (String string) {
-	if (string == null) {
+
+	/* If the string is empty, return the string. */
```

Comment:
Also a premature optimization that could be left away

Reply:
Done